static/css/dark.css

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
Can you make the links in the 'dark' theme white instead of orange ?

Reply:
Really? 

I changed the text to be white, so white links + white text seems a bit weird.

If you are sure, I will change it.

Reply:
I should have been a bit more specific: When you look at the current monitor front page (http://138.197.179.164/) text color is light gray with white links, except for the "tables" with the node infos, where all is white - text and links. 

In regular text, there should be a color difference between text and links. You are right on that! I feel that the orange links are a bit too much, so I'd prefer gray vs. white. In the are where the actual node infos are being displayed, I would not want to have a color difference between text and links. 

Having the text in light gray in the current version makes the main info, the node stats, stick out a bit more in my opinion. But I am also fine with white text and maybe light gray links, i.e. switch it around. 

Let's wait for @BitDesert 's view on that ...